schema.graphql

```diff
@@ -8,3 +8,12 @@ type BuyBack @entity {
   dai: BigInt!
   yfi: BigInt!
 }
+
+type BuyBackContract @entity {
```

Comment:
Question:

I wonder if we need to store history (and tx hash for tracking purposes). I guess it not required, but just in case.